flask_admin/model/base.py

```diff
@@ -853,10 +854,18 @@ def scaffold_form(self):
         """
         raise NotImplementedError('Please implement scaffold_form method')
 
-    def scaffold_list_form(self):
+    def scaffold_list_form(self, CustomFieldList, validators=None):
```

Comment:
Is it possible to provide default value for the `CustomFieldList`? Also, lets use underscore variable name even if it is a class value.